signup/signup.css

```
@@ -0,0 +1,156 @@
+*{
+    box-sizing: border-box;
+}
+
+body{
+    background: var(--Linkbrary-bg);
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 30px;
+
+    margin: 0;
+    padding-top: 238px;
+    padding-bottom: 252px;
+}
+
+div.page-logo{
+    margin-bottom: 16px;
+}
+
+div.page-logo img{
+    width: 210.583px;
+    height: 38px;    
+}
+
+div.signup-text{
+    display: flex;
+    gap: 8px;
+}
+
+div.signup-text .signin-link{
+    padding-bottom: 5px;
+    
+    color: var(--Linkbrary-primary-color);
+    font-family: Pretendard;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;    
+}
+
+form.signup-form{
+    display: flex;
+    flex-direction: column;
+}
+
+form.signup-form label{
+    color: var(--black);
+    font-family: Pretendard;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;    
+}
+
+form.signup-form input{
+    width: 370px;
+    padding: 18px 15px;
+    border-radius: 8px;
+    border: 1px solid var(--Linkbrary-primary-color-2);
+    background: var(--Linkbrary-white);    
+    margin-top: 12px;
+    margin-bottom: 24px;
+}
+
+form.signup-form input:focus{
+    border: 1px solid var(--Linkbrary-primary-color);
+}
+
+form.signup-form input#password-check-box{
+    margin: 0;
```

하단의 간격을 없애기 위해 사용하신것 같은데요. 이 코드로 인해 label과 input 사이의 간격도 0으로 되어서 딱 붙어보이는 버그가 발생한것 같아요! 이때는 하단 간격만 0으로 변경하는 코드의 사용을 제안합니다.
```suggestion
    margin-bottom: 0;
```